repology/parsers/parsers/debian.py

```diff
@@ -131,6 +131,10 @@ def _extra_handling(self, pkg: PackageMaker, pkgdata: dict[str, str]) -> None:
             pkg.add_name(pkgdata['SourceName'], NameType.OPENWRT_SOURCENAME)
         pkg.set_arch(pkgdata['Architecture'])
         pkg.set_extra_field('path', '/'.join(pkgpath[2:]))
+        pkg.add_homepages(pkgdata.get('URL'))
+        if cpeid := pkgdata.get('CPE-ID'):
+            cpe_components = cpeid.split(':')
+            pkg.add_cpe(cpe_components[3], cpe_components[4])
```

Comment:
Also there's a single case of broken CPE-ID, 
```
CPE-ID: cpe:/a:lrzsz_project
```
from lrzsz in openwrt 2022. Can this be fixed in the repository or do we need a workaround here?

Reply:
Please feel free to drop 22.03 since it's no longer build. Even if I'd backport that patch, there is no build infrastructure to create new packages.